Sources/TuistGraph/Graph/GraphOutputJSON.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+import TSCBasic
+
+public struct GraphOutputJSON: Codable, Equatable {
+    public var name: String
```

Comment:
is there a specific reason for having all the properties as variables? I'm not sure that's completely necessary.

Reply:
Good point. We'll make all of them `let`.